doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -8,4 +8,4 @@ This document provides information to deploy the various CORTX OVA releases:
 # For OVA 2.0.0
 
 - To know about new features, enhancements, and fixed issues, see `Release Notes <ova/2.0.0/CHANGELOG-2.0.0.md>`_.
```

Comment:
this is the latest wiki page for changelog, https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-4/CHANGELOG.md